grails-bootstrap/src/main/groovy/org/grails/exceptions/reporting/DefaultStackTracePrinter.groovy

```diff
@@ -23,7 +23,7 @@ import org.codehaus.groovy.control.MultipleCompilationErrorsException
  */
 class DefaultStackTracePrinter implements StackTracePrinter {
 
-    String prettyPrint(Throwable t) {
+    String prettyPrint(Throwable t, Map attrs = [:]) {
```

Comment:
Using Groovy default params on overridden/implemented methods makes the code hard to follow. Can we use:
```groovy
@Override
String prettyPrint(Throwable t) {
    prettyPrint(t, [:])
}

String prettyPrint(Throwable t, Map attrs) {
    
}
```

(or change the interface to include both versions)